standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestHiveMetaStoreTimeout.java

```diff
@@ -43,9 +46,10 @@ public class TestHiveMetaStoreTimeout {
   protected static HiveMetaStoreClient client;
   protected static Configuration conf;
   protected static Warehouse warehouse;
+  protected static int port;
 
-  @BeforeClass
-  public static void setUp() throws Exception {
+  @Before
+  public void setUp() throws Exception {
     HMSHandler.testTimeoutEnabled = true;
```

Comment:
Nit: This test code hacks `HMSHandler` class, I will refactor this in a new PR.

Reply:
do we need to repeat the whole setup for every test or it's possible to extract just the HMSHandler part?

Reply:
Addressed! Have separated the start of MetaStore Server into a `BeforeClass` method.